teamBangazon/BangazonAPI
# Customer resource

###Changes Made
Added Customer Resource Dev Access

###Steps to Test
1. Once code is pulled down make sure that migrations folder and existing local DB have been deleted.
2. Run: dotnet restore 
 - then run: dotnet ef migrations add initial
 - then run: dotnet ef database update
 - then: dotnet run
3. Once api is running in postman POST to http://localhost:5000/api/customer/  - making sure to set content-type:application/json in the header.  The body only needs to contain FirstName and LastName keys and Values.
4. Once a customer has been successfully posted a GET should pull a list of all customers added.
5. PUT should work similarly.  Make sure to include the full edited object in the body and designate the customerId at the end of the URL like this: http://localhost:5000/api/customer/1 
###Ref URL
https://github.com/teamBangazon/BangazonAPI/issues/1
